bokeh/plotting/figure.py

```diff
@@ -83,6 +83,10 @@ class FigureOptions(Options):
     The type of the y-axis.
     """)
 
+    tooltips = Either(String, List(Tuple(String, String)), help="""
+
```

Comment:
Missing tooltip.

Reply:
Thanks, I have added it and will merge when green.